### ISSUE TITLE
Worksheet Copy throws System.NullReferenceException

### ISSUE THREAD

**Issue by connorfcm** [EPPlusSoftware/EPPlus#341]
[Base_debug.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/6205209/Base_debug.xlsx)
[Test_debug.xlsx](https://github.com/EPPlusSoftware/EPPlus/files/6205240/Test_debug.xlsx)

Worksheet Copy causes System.NullReferenceException. The attached Excel files have been stripped down to only the relevant features. Looks like the Exception has something to do with an underlying CopyDrawing() call.

```
[Fact]
public void EPPlusBug()
{
    ExcelPackage.LicenseContext = LicenseContext.Commercial;
    var x = @"D:\Path\to\Base_debug.xlsx";
    var y = @"D:\Path\to\Test_debug.xlsx";
    using (var package = new ExcelPackage(new FileInfo(x)))
    {
        using (var atomic_sheet_package = new ExcelPackage(new FileInfo(y)))
        {
            var s = atomic_sheet_package.Workbook.Worksheets["Test3"];
            var s_copy = package.Workbook.Worksheets.Add("Test3", s); // Exception on this line
            atomic_sheet_package.Save();
        }
        package.Save();
    }
}
```

**Comment by JanKallman** [comment]
This seems to happend as the XSeries property is blank. I will investigate more an provide a fix.

**Comment by JanKallman** [comment]
Fixed in 206387b96fe746da0264d7dcce6606ec9e6a910a 
Feel free to try it out using our develop nuget feed. 
See https://github.com/EPPlusSoftware/EPPlus/wiki/Using-our-Develop-Nuget-Feed

**Comment by connorfcm** [comment]
Just verified the fix on the develop nuget feed! Great work, thanks!

**Comment by JanKallman** [comment]
Fixed in version 5.6.2
